sites/public/lib/appAutofill.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Address,
+  Application,
+  ApplicationStatus,
+  ApplicationSubmissionType,
+  Language,
+} from "@bloom-housing/backend-core/types"
+
+class AutofillCleaner {
+  application: Application = null
+
+  constructor(application: Application) {
+    this.application = application
+  }
+
+  clean() {
+    // prettier-ignore
+    this.
+      addDefaults().
+      removeAdditionalKeys().
+      removeLiveWorkAddresses()
+
+    return this.application
+  }
+
+  addDefaults() {
+    ;["id", "createdAt", "updatedAt", "deletedAt", "listing", "submissionDate"].forEach((key) => {
```

Comment:
`;` <- I suppose it's not necessary :D

Reply:
Prettier is adding it…must have thought there could be a JS error otherwise.